package.json

```diff
@@ -29,7 +29,7 @@
     "@types/ws": "^3.2.1",
     "babel-cli": "^6.26.0",
     "babel-preset-es2015": "^6.24.1",
-    "browserify": "^14.3.0",
+    "browserify": "^16.5.0",
```

Comment:
@makarandp0 I think the most recent browserify version is 17. Would be good to update.